songDisplayComponent/public/style.css

```diff
@@ -0,0 +1,174 @@
+html {
+  background-color: #cccccc;
+}
+
+body {
+  font: 12px/1.4 'Lucida Grande', 'Lucida Sans Unicode', 'Lucida Sans', Garuda,
+    Verdana, Tahoma, sans-serif;
+  margin: 0;
+  grid-template-rows: [row1] 30px [row2] auto;
+  height: 1500px;
+}
+
+.container {
+  width: 70%;
+  margin: 0 15%;
+}
+
+.nav-bar {
+  background-image: url('../public/Assets/navbar.png');
+  height: 46px;
+}
+
+.outer-player-panel {
+  margin: 0 20px 0 20px;
+  height: 380px;
+  min-width: 900px;
+  grid-row-start: row2;
+}
+
+.inner-player-panel {
+  height: 100%;
+  width: 100%;
+  display: inline-grid;
+  grid-template-columns: [col1] 30px [col2] auto [col3] 340px [col4] 20px;
+  grid-template-rows: [row1] 30px [row2] 70px [row3] 280px [row4] 100px [row5] 20px;
+  background: linear-gradient(135deg, rgb(149, 91, 78) 0%, rgb(72, 55, 59) 100%)
+    repeat scroll 0% 0%;
+}
+
+.player-head {
+  display: inline-grid;
```

Comment:
nice job using css grid.

Reply:
I like your thinking on separating the DB connection from the query methods -- I'll include that in my next PR